core/translate/plan.rs

```diff
@@ -1,6 +1,9 @@
 use core::fmt;
 use limbo_ext::{ConstraintInfo, ConstraintOp};
-use limbo_sqlite3_parser::ast::{self, SortOrder};
+use limbo_sqlite3_parser::{
+    ast::{self, SortOrder},
+    to_sql_string::{ToSqlContext, ToSqlString},
+};
```

Comment:
Could everything related to `to_sql_string` be in a separate module? In fact we could move the Display implementations for `Plan` there too to reduce noise wrt. the core functionality